WooCommerce/Classes/System/SessionManager.swift

```diff
@@ -21,10 +21,15 @@ extension NSNotification.Name {
     public static let applicationTerminating = Foundation.Notification.Name(rawValue: "ApplicationTerminating")
 }
 
+private extension UserDefaults {
+    @objc dynamic var defaultStoreID: Int {
+        integer(forKey: Key.defaultStoreID.rawValue)
```

Comment:
Did you mean opening this up directly for whoever interested in the default store ID to subscribe? `UserDefaults` seems an implementation detail of `SessionManager`, and that's why I created a public observable `siteID: Observable<Int64?>` from `UserDefaults` KVO. Also, `UserDefaults` KVO has some limitation since it's ObjC API and doesn't work with `Int64` directly.

Not sure if I misunderstood anything, feel free to suggest another way!